05/app.js

```diff
@@ -1,3 +1,54 @@
-console.log('DOM');
+console.log("DOM");
 
-const curr = document.querySelector('.js-curr');
+const curr = document.querySelector(".js-curr");
+console.log(curr);
+
+// Utwórz kolejny przycisk, który będzie rodzeństwem (bratem) dla wyszukanego w zmiennej curr elementu. Element ten niech zawiera napis usuń z koszyka.
+
+const deleteButton = document.createElement("button");
+deleteButton.innerHTML = "Usuń z koszyka";
+curr.parentElement.appendChild(deleteButton);
+
+// dla wszystkich elementów, które są rodzeństwem dla elementu o klasie .js-curr dodaj klasę .siblings (wykorzystaj pętlę)
+
+const siblingList = [];
+let sibling = curr.parentNode.firstChild;
+while (sibling) {
+  if (sibling.nodeType === 1 && sibling !== curr) {
+    siblingList.push(sibling);
+  }
+  sibling = sibling.nextSibling;
+}
+
+// dlaczego musimy sprawdzic czy sibling nie jest nodeType === 1 ?
```

Comment:
Mam różne type 'nodes': https://www.w3schools.com/jsref/prop_node_nodetype.asp dlatego musisz to sprawdzić.
Gdybyś użyła "firstElementChild" to od razu byś miała "Element" i problem by nie występował ;)

Reply:
Dziekuje za link :) Teraz juz bede wiedziec o co chodzi w starszych rozwiazaniach :) Poprawilam na element i zostawilam sobie stara wersje, zeby moc sobie przypomniec :)